### ISSUE TITLE
Brands for paltes

### ISSUE THREAD

**Issue by Dzi4d3k** [TeamFSIModding/FS17_Goweil_LT_Master#27]
I see that you changed picture of additives palette in shop. I have suggestion to change also brand of this palette like it is done in Mengele Garant 540/2 form Stefan Maurus
http://www.farming-simulator.com/mod.php?lang=en&country=gb&mod_id=50217
for the same kind of palette.

**Comment by fcelsa** [comment]
Yes we know... in helpline I need a new screen for pallet, I'm working on it, for final version in the shop, we think about brand, but maybe Lizard it's more sure for the Giants rules.

**Comment by Dzi4d3k** [comment]
As far as I remember there can't be any non licensed brands on consoles, yet look at Mengele Garant it is on consoles too, but Stefan Maurus is very close to Giants, so this could be an answer.
Interesting is also that trailer contain custom scripts (as your LT Master) and it is con consoles.

**Comment by fcelsa** [comment]
Yes we know, Maurus work for Giants also, I think not all functions of Mengele is on console, due to limitations, but I'm not sure.
